src/api/index.tsx

```diff
@@ -255,7 +266,10 @@ export const getVersionsList = async ({
 };
 
 export const logOutFromServer = async (apiState: ApiState) => {
-  return callApi<GetResource<{}>>({
+  return callApi<{
+    requestData: undefined;
+    successfulResponse: { ok: boolean };
+  }>({
```

Comment:
Shouldn't this be `GetResource<{ok: boolean}>`?

Reply:
I guess it could be then since it's not a `GET` I had second thoughts ... then I couldn't think of a better name for `GetResource`. `EmptyRequestResponse` seems weird, heh. Maybe I'll think of one.

Reply:
Maybe `GetResource` could be renamed to `Resource`?

Reply:
Hmm, seems sort of vague. I went with `ResponseOnly`, let me know what you think.

Reply:
Works for me

Reply:
Work for me

Reply:
Works for me